crates/core/tedge/src/cli/connect/command.rs

```diff
@@ -74,9 +74,9 @@ impl Command for ConnectCommand {
             let br_config = self.bridge_config(&config)?;
             if self.check_if_bridge_exists(&br_config) {
                 return match self.check_connection(&config) {
-                    Ok(DeviceStatus::AlreadyExists) => {
-                        let cloud = br_config.cloud_name.clone();
-                        println!("Connection check to {} cloud is successful.\n", cloud);
+                    Ok(_) => {
+                        let cloud = br_config.cloud_name;
+                        println!("Connection check to {} cloud is successfull.\n", cloud);
```

Comment:
Previous spelling was correct :)
```suggestion
                        println!("Connection check to {} cloud is successful.\n", cloud);
```

Reply:
Whoopsie, that slipped. Thanks!